### PR TITLE
Here's the CSharp version

### PR DESCRIPTION
**Pull request by taranasus**
Hi Dragon, here's the C# Version!

Good luck figuring out how to build it on Linux

### CODE REVIEW

File: CSharp/CLI_Calc_V1_0/Program.cs [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+                case 3:
+                    result = a * b;
+
+                    break;
+                case 4:
+                    result = a / b;
+                    break;
+                case 5:
+                    // I know you said no built-ins but nobody on this planet would EVER not use the 
+                    // C# Math class to do Math operations. EVER.
```
Comment (Dragonsight91) [inline CSharp/CLI_Calc_V1_0/Program.cs:69]: fair point, you could use a simple iterative solution, but i guess this works too.

File: CSharp/CLI_Calc_V1_0/Program.cs [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+                Console.Write($"{variableName}: ");
+            }
+
+            return a;
+        }
+
+        // This method is a bit fancier, it returns what's called a touple, which basically allows for the return of
+        // multiple variables in C# and is quicker than creating a whole Model Class, but only good for two or three variables
+        // otherwise it makes more sense to create a Model Class and return an object
+        public static (string functionName, string functionExpression) GetFunctionNameAndExpression(int function)
```
Comment (Dragonsight91) [inline CSharp/CLI_Calc_V1_0/Program.cs:117]: i mean this is one way to do it. not a bad one, just something i didn't expect to see

**Review by Dragonsight91 — CHANGES_REQUESTED** [review]
There's only one change that i wanna request and that is following the naming convention/project folder convention
the actual program folder structure should be in a folder called `Calc1` so that this can be one of many versions submitted. As i alluded to in one of my comments, there were some unexpected things, that i would've done differently.
see current version of `README.md` for more details.

That's it, code is tested and works, it is within spec +/- an anomaly, not that i really expected any different, but the code itself is approved.

**Comment by taranasus** [comment]
Folder structure updated

**Review by Dragonsight91 — APPROVED** [review]
Requested change applied, PR approved.

**Comment by taranasus** [comment]
YEY!

**Pull request merged**